incubator/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.15.1
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.13.4
+version: 0.14.0
```

Comment:
There is already another PR which is approved and has the same version.

Reply:
Yes, making 1 logical change per PR now :)
Was not sure which would merge first, will update now, thanks!